runbotics/runbotics-orchestrator-ui/src/src-app/components/tables/NotificationTable/useProcessNotificationColumns.tsx

```diff
@@ -0,0 +1,62 @@
+import DeleteIcon from '@mui/icons-material/Delete';
+import {
+    GridActionsCellItem,
+    GridEnrichedColDef,
+    GridRowParams,
+    GridValueFormatterParams,
+} from '@mui/x-data-grid';
+
+import moment from 'moment';
+
+import useTranslations from '#src-app/hooks/useTranslations';
+
+import { ProcessNotificationRow } from './NotificationTableComponent.types';
+import { SubscribersTableFields } from './NotificationTableComponent.utils';
+
+interface ColumnsActions {
+    onDelete: (processSubscriber: ProcessNotificationRow) => Promise<void>;
+}
+
+const useProcessNotificationColumns = ({
+    onDelete,
+}: ColumnsActions): GridEnrichedColDef[] => {
+    const { translate } = useTranslations();
+
+    return [
+        {
+            field: SubscribersTableFields.USER,
+            headerName: translate('Process.Edit.Table.Columns.User'),
+            flex: 0.4
+        },
+        {
+            field: SubscribersTableFields.SUBSCRIBED_AT,
+            headerName: translate('Process.Edit.Table.Columns.SubscribedAt'),
+            flex: 0.4,
+            valueFormatter: (params: GridValueFormatterParams) =>
+                moment(params.value as string).format('YYYY-MM-DD HH:mm'),
+        },
+        {
+            field: SubscribersTableFields.ACTIONS,
+            headerName: translate('Process.Edit.Table.Columns.Actions'),
+            type: 'actions',
+            flex: 0.2,
+            getActions: (params: GridRowParams<any>) => {
```

Comment:
this is veeeeery similar set to the useBotNotificationColumns.
consider creating a function that takes only headerNames as arguments